<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatProgram</class>
 <widget class="QMainWindow" name="ChatProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>1112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Program - V1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>1061</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Server_Tab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QToolButton" name="Start_Button">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QToolButton" name="Stop_Button">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QComboBox" name="IP_ComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="IP_Label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HY나무B</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QComboBox" name="PORT_ComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Message_Browser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>751</width>
        <height>781</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="PORT_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HY나무B</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PORT</string>
      </property>
     </widget>
     <widget class="QLabel" name="LOGO_Label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>97</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>LOGO MESSAGE</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Time_Browser">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Agency FB'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="LOGO_Label_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>100</y>
        <width>48</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>USERS</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="UserList_Browser">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>130</y>
        <width>221</width>
        <height>781</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Agency FB'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QToolButton" name="Trans_Button">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>920</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>전송</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Broadcase_Edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>920</y>
        <width>751</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Client_Tab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QTextBrowser" name="Cli_Message_Browser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>751</width>
        <height>781</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="Entere_Button">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Cli_PORT_ComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Cli_UserList_Browser">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>130</y>
        <width>221</width>
        <height>781</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Agency FB'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cli_IP_Label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>32</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HY나무B</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Cli_Time_Browser">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Agency FB'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Cli_IP_ComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Cli_PORT_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HY나무B</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PORT</string>
      </property>
     </widget>
     <widget class="QToolButton" name="Cli_Trans_Button">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>920</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>전송</string>
      </property>
     </widget>
     <widget class="QToolButton" name="Exit_Button">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cli_Message_Label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>69</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>MESSAGE</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cli_LOGO_Label">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>100</y>
        <width>48</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>USERS</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Cli_Chat_Edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>920</y>
        <width>751</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>34</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>HY나무B</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Open"/>
    <addaction name="Save"/>
    <addaction name="SaveAs"/>
    <addaction name="separator"/>
    <addaction name="Print"/>
    <addaction name="separator"/>
    <addaction name="Settings"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="font">
     <font>
      <family>HY나무B</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="Undo"/>
    <addaction name="Redo"/>
    <addaction name="separator"/>
    <addaction name="Cut"/>
    <addaction name="Copy"/>
    <addaction name="Paste"/>
    <addaction name="Delete"/>
    <addaction name="SelectAll"/>
    <addaction name="separator"/>
    <addaction name="Find"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="font">
     <font>
      <family>HY나무B</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="Help"/>
    <addaction name="separator"/>
    <addaction name="About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Open">
   <property name="text">
    <string>Open                   Ctrl + O</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Save                    Ctrl + S</string>
   </property>
  </action>
  <action name="SaveAs">
   <property name="text">
    <string>Save As      Ctrl + Shift + S</string>
   </property>
  </action>
  <action name="Print">
   <property name="text">
    <string>Print                    Ctrl + P</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="Undo">
   <property name="text">
    <string>Undo                   Ctrl + Z</string>
   </property>
   <property name="font">
    <font>
     <family>HY나무B</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="Settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="Redo">
   <property name="text">
    <string>Redo         Ctrl + Shift + Z</string>
   </property>
  </action>
  <action name="Cut">
   <property name="text">
    <string>Cut                      Ctrl + X</string>
   </property>
  </action>
  <action name="Copy">
   <property name="text">
    <string>Copy                    Ctrl + C</string>
   </property>
  </action>
  <action name="Paste">
   <property name="text">
    <string>Paste                    Ctrl + P</string>
   </property>
  </action>
  <action name="Delete">
   <property name="text">
    <string>Delete                   Delete</string>
   </property>
  </action>
  <action name="Find">
   <property name="text">
    <string>Find                      Ctrl + F</string>
   </property>
  </action>
  <action name="SelectAll">
   <property name="text">
    <string>Select all                Ctrl + A</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
